src/App.js

```diff
@@ -41,13 +43,21 @@ function App() {
             <Route
               exact
               path="/"
-              render={() => <Welcome setToken={setToken} />}
+              render={() =>
+                token ? (
+                  <Redirect to="/list" />
+                ) : (
+                  <Welcome setToken={setToken} />
+                )
+              }
```

Comment:
@katmdev You could create an issue in the github project for that!

Reply:
@katmdev I agree that it would be great if a user could access more than one list!  I think we may have brought this up in another meeting and someone mentioned that it would be a stretch feature past the scope of the current project.  Perhaps we could add that in as another task?  I like that it redirects away from the welcome page if there is a token set in local storage, but maybe we could have an "exit list" button that would clear local storage and bring the user back to the welcome page.

Reply:
@laurenyz I've added a new issue (#29 ) to create an exit list button with the redirect back to the welcome page as per your comment and also added the request for the token to be accessible to users.

Reply:
Great catch and resolution!